### ISSUE TITLE
Publish ESRi, Oracle modules on Maven

### ISSUE THREAD

**Issue by andyjefferson** [locationtech/jts#439]
These have non-open source dependencies but, unless they are published, they will not be useable (without inflicting on developers to "go off and build it yourself"). 

In the case of Oracle the only additional work required (by the Maven releaser) is to install an Oracle JDBC driver jar in their local Maven repo, and then build/release the JTA module (in a suitable place in Maven).

FWIW Vividsolutions managed to publish such modules without apparent problem.

**Comment by mprins** [comment]
> In the case of Oracle the only additional work required (by the Maven releaser) is to install an Oracle JDBC driver jar in their local Maven repo, and then build/release the JTS module (in a suitable place in Maven).

Actually to run (at least some of) the testcases a developer also needs to have an oracle instance running

**Comment by yeroc** [comment]
This is a serious pain.  Now that Oracle drivers are in Maven Central is there any reason this can't be done?

**Comment by mprins** [comment]
@yeroc @andyjefferson Creating a PR that updates `https://github.com/locationtech/jts/blob/master/modules/io/ora/pom.xml` and `.travis.yml` to build the oracle module could be a start

**Comment by dr-jts** [comment]
> Actually to run (at least some of) the testcases a developer also needs to have an oracle instance running

Yes, that is a serious impediment to maintaining and publishing these modules, IMO.  I am no longer in a position to run the tests for these modules . I suppose the Oracle module could be released in a "compile-only" way, if that is worth anything. The unit tests would have to be disabled in some way for CI, I think?

Other ideas welcome.

**Comment by dr-jts** [comment]
> FWIW Vividsolutions managed to publish such modules without apparent problem.

At Vivid we had an environment in which to develop and run the tests.

**Comment by andyjefferson** [comment]
FWIW Oracle JDBC driver(s) are in Maven central. 

Change https://github.com/locationtech/jts/blob/master/pom.xml#L116
to be "ojdbc8" and version 12.2.0.1

Change https://github.com/locationtech/jts/blob/master/modules/io/ora/pom.xml#L40
to be "ojdbc8".

Change https://github.com/locationtech/jts/blob/master/modules/io/ora/README.md
to remove the lines about needing an Oracle JDBC driver, since one is in the Maven repo.

That *presumably* would anyone to build it. You'd have to disable tests without a working Oracle instance, clearly. 

I tried "mvn clean install -DskipTests" followed by "mvn install -Poracle -DskipTests" but got some Maven whinges on the second command about "Failure to find org.locationtech.jts:jts-io:pom:1.18.0-SNAPSHOT". I'll leave it to someone who understands your Maven hierarchy to work that one out.
